File: tests/fixtures/codegen/expected/typescript/runtime/queries/admin/events/GetEvent_linux.ts
import type { GetEventLinuxParams, GetEventLinuxRow } from "../../../../types/queries/admin/events/GetEvent_linux.js";
export type { GetEventLinuxParams, GetEventLinuxRow } from "../../../../types/queries/admin/events/GetEvent_linux.js";
import type * as pgsid from "./pgsid/queryable.js";
export const getEventLinuxSql = "SELECT 1 AS value;";
export async function getEventLinux(db: pgsid.Queryable): Promise<GetEventLinuxRow | undefined> {
    const result = await db.query(getEventLinuxSql, []);
    const row = result.rows[0];
    if (row === undefined)
        return undefined;
    return row as GetEventLinuxRow;
}

File: tests/fixtures/codegen/expected/go/queries/admin/events/geteventlinux.go
package events

import "context"

const GetEventLinuxSQL = "SELECT 1 AS value;"

type GetEventLinuxParams struct {
}
type GetEventLinuxRow struct {
	Value int32 `db:"value"`
}

func (q *Queries) GetEventLinux(ctx context.Context) (GetEventLinuxRow, error) {
	var row GetEventLinuxRow
	err := q.db.QueryRow(ctx, GetEventLinuxSQL).Scan(&row.Value)
	return row, err
}
